devenv.nix

```
@@ -126,54 +130,49 @@ let
   '';
 
   importDbHelper = pkgs.writeScript "importDbHelper" ''
-    PATH="${lib.makeBinPath [ pkgs.coreutils ]}:$PATH"
-
     if [[ "$1" == "" ]]; then
         echo "Please set devenv configuration for kellerkinder.importDatabaseDumps"
         exit
     fi
 
-    if ! ${config.services.mysql.package}/bin/mysqladmin ping > /dev/null 2>&1; then
+    if ! $DEVENV_PROFILE/bin/mysqladmin ping > /dev/null 2>&1; then
```

```suggestion
    if ! "${config.env.DEVENV_PROFILE}/bin/mysqladmin" ping > /dev/null 2>&1; then
```
Shouldn't it be more like this?